person.rb

```diff
@@ -20,6 +21,10 @@ def validate_name(_name)
     @name = @correct_name.correct_name(@name)
   end
 
+  def add_rental(rental)
+    rentals.push(rental)
+  end
```

Comment:
- [ ] Let's modify this method and ensure that the person itself is sent as a parameter to create the new rental achieving the 'has-many' association. And also to avoid duplicates. See suggestion below:

```ruby
def add_rental(book, date)
    # *The person itself should be sent as a parameter to create the new rental achieving the 'has-many' association
    Rental.new(date, book, self)
  end
```